web-packages/common/package.json

```diff
@@ -4,46 +4,30 @@
     "version": "0.0.0",
     "type": "module",
     "scripts": {
-        "build_wasm": "bash ../reti-js/init_test.sh",
+        "build_wasm": "bash ../../reti-js/init_test.sh",
         "dev": "pnpm build_wasm && vite",
         "build": "pnpm build_wasm && vite build",
         "preview": "pnpm build_wasm && vite preview",
         "check": "pnpm build_wasm && svelte-check --tsconfig ./tsconfig.json && tsc -p tsconfig.node.json",
-        "lint": "pnpm build_wasm && eslint './src/**/*.{js,ts,svelte}'",
         "test": "pnpm build_wasm && vitest run",
-        "test:watch": "pnpm build_wasm && vitest",
-        "fmt": "prettier . --write",
-        "fmt:check": "prettier . --check"
+        "test:watch": "pnpm build_wasm && vitest"
     },
     "devDependencies": {
-        "@eslint/js": "^9.7.0",
         "@sveltejs/vite-plugin-svelte": "^3.1.1",
         "@testing-library/jest-dom": "^6.4.6",
         "@testing-library/svelte": "^5.2.0",
         "@tsconfig/svelte": "^5.0.4",
         "@types/katex": "^0.16.7",
-        "eslint": "^9.7.0",
-        "eslint-plugin-svelte": "^2.43.0",
-        "globals": "^15.8.0",
         "happy-dom": "^14.12.3",
-        "prettier": "3.3.3",
-        "prettier-plugin-svelte": "^3.2.6",
         "svelte": "^4.2.18",
         "svelte-check": "^3.8.4",
         "tslib": "^2.6.3",
-        "typescript": "^5.5.3",
-        "typescript-eslint": "8.0.0-alpha.44",
         "vite": "^5.3.4",
         "vite-plugin-top-level-await": "^1.4.1",
         "vite-plugin-wasm": "^3.3.0",
         "vitest": "^2.0.3"
     },
     "dependencies": {
         "katex": "^0.16.11"
-    },
-    "pnpm": {
-        "override": {
-            "@swc/core": "~1.6.13"
```

Comment:
I tried running this and it is working but i really dont understand why.
Looking at pnpm output it says that it is using the broken swc core 1.7.0 but it works on your and my machine so it is fine. Just really wierd lol.

Reply:
Before I removed this I got a warning that says "this pnpm override has no effect" because I needed to put it in the root package.json instead, so I think pnpm should still use the fixed version because its overridden in root.